tests/util/test_full_block_utils.py

```diff
@@ -204,9 +207,9 @@ def get_full_blocks():
 
     for foliage in get_foliage():
         for foliage_transaction_block in get_foliage_transaction_block():
-            height = random.randint(0, 1000000)
+            height = random.randint(0, 100000)
```

Comment:
why did you make this change?

Reply:
shouldn't be added, this was left from testing, will remove